prac_05/emails.py

```diff
@@ -0,0 +1,32 @@
+"""
+Emails
+Estimate: 30 minutes
+Actual: 50 minutes
+"""
+
+email = str(input("Email: "))
+email_to_name = {}
+
+while email != "":
+    # decipher user's name
```

Comment:
This would be great as a function. Maybe decipher_username() ?
Also be sure to make your comment into the docstring, and capitalise that first word 😁😎